src/main/scala/io/iohk/ethereum/blockchain/sync/PeersClient.scala

```diff
@@ -37,7 +37,18 @@ class PeersClient(
   def running(requesters: Requesters): Receive =
     handleBlacklistMessages orElse handlePeerListMessages orElse {
       case PrintStatus =>
-        log.debug("PeersClient status: requests: {}, available peers: {}", requesters.size, peersToDownloadFrom.size)
+        log.debug("PeersClient status: requests in progress: {}, available peers: {}", requesters.size, peersToDownloadFrom.size)
+
+        log.debug(s"Reporting status of our ${handshakedPeers.size} handshaked peers")
+        handshakedPeers.foreach {
+          case (peer, info) =>
+            val peerNetworkStatus = PeerNetworkStatus(
+              peer,
+              isBlacklisted = isBlacklisted(peer.id)
+            )
+            log.debug(s"${peer.id} - $peerNetworkStatus")
```

Comment:
I would prefer to put everything in one log msg. It will be easier to debug (when we will have structured logging) to have the whole list in one log entry